README.md

```diff
@@ -140,6 +140,9 @@ vars:
     accounts_pass_through_columns: 
         - name: "new_custom_field"
           alias: "custom_field"
+    accounting_periods_pass_through_columns:
```

Comment:
We will want to make a similar update re: this new variable in the `dbt_netsuite` README.

Reply:
Yeah I was wondering about this, but I noticed that the transform [README](https://github.com/fivetran/dbt_netsuite/tree/main?tab=readme-ov-file#passing-through-additional-fields) doesn't include _any_ passthrough columns that aren't persisted past the staging layer (`vendors_pass_through_columns`, `items_pass_through_columns`, `entities_pass_through_columns`).

So the question is, do we wanna add all of them to the transform README (and note that they're not included in any end models)? I lean toward yes

Reply:
Would lean toward yes here. Should definitely let customers know of this functionality.